language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/codeaction/CodeActionTest.java

```diff
@@ -237,42 +237,48 @@ public void testCodeActionWithQuickFixes(String config, String source)
-                {"markUntaintedCodeAction1.json", "taintedVariable.bal"},
-                {"markUntaintedCodeAction2.json", "taintedVariable.bal"},
-                {"variableAssignmentRequiredCodeAction1.json", "createVariable.bal"},
-                {"variableAssignmentRequiredCodeAction2.json", "createVariable.bal"},
-//                {"variableAssignmentRequiredCodeAction3.json", "createVariable.bal"},
-                {"variableAssignmentRequiredCodeAction4.json", "createVariable.bal"},
-                {"variableAssignmentRequiredCodeAction5.json", "createVariable2.bal"},
-                {"variableAssignmentRequiredCodeAction6.json", "createVariable2.bal"},
-                {"variableAssignmentRequiredCodeAction7.json", "createVariable2.bal"},
-                {"variableAssignmentRequiredCodeAction8.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction9.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction10.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction11.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction12.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction13.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction14.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction15.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction16.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction17.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction18.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction19.json", "createVariable3.bal"},
-                {"variableAssignmentRequiredCodeAction20.json", "createVariable4.bal"},
-                {"variableAssignmentRequiredCodeAction21.json", "createVariable4.bal"},
-                {"variableAssignmentRequiredCodeAction22.json", "createVariable4.bal"},
-                {"variableAssignmentRequiredCodeAction23.json", "createVariable4.bal"},
-                {"variableAssignmentRequiredCodeAction24.json", "createVariable4.bal"},
-                {"variableAssignmentRequiredCodeAction25.json", "createVariable4.bal"},
-                {"variableAssignmentRequiredCodeAction26.json", "createVariable4.bal"},
-                {"variableAssignmentRequiredCodeAction27.json", "createVariable4.bal"},
-                {"variableAssignmentRequiredCodeAction28.json", "createVariable4.bal"},
-                {"variableAssignmentRequiredCodeAction29.json", "createVariable5.bal"},
-                {"variableAssignmentRequiredCodeAction30.json", "createVariable5.bal"},
-                {"variableAssignmentRequiredCodeAction31.json", "createVariable5.bal"},
+//                {"fixReturnType1.json", "fixReturnType.bal"},
```

Comment:
Any specific reason to disable these test cases?

Reply:
That was done for the testing purposes. Will fix it.